File: backend/app/Models/Package.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\BelongsTo;
use Illuminate\Database\Eloquent\Relations\HasMany;

class Package extends Model
{
    use HasFactory;

    protected $fillable = [
        'category_id',
        'name',
        'name_ar',
        'slug',
        'logo_url',
        'sort_order',
        'is_active',
    ];

    protected $casts = [
        'is_active' => 'boolean',
        'sort_order' => 'integer',
    ];

    public function category(): BelongsTo
    {
        return $this->belongsTo(Category::class);
    }

    public function channels(): HasMany
    {
        return $this->hasMany(Channel::class)->orderBy('sort_order');
    }
}

File: android/app/src/main/java/com/sportiptv/app/domain/model/Package.kt
package com.sportiptv.app.domain.model

data class Package(
    val id: Long,
    val categoryId: Long,
    val name: String,
    val nameAr: String?,
    val slug: String,
    val logoUrl: String?,
    val sortOrder: Int,
    val isActive: Boolean,
    val channelsCount: Int
)
